src/measure_theory/tactic.lean

```diff
@@ -0,0 +1,179 @@
+/-
+Copyright (c) 2021 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import tactic.auto_cases
+import tactic.tidy
+import tactic.with_local_reducibility
+import tactic.show_term
+import measure_theory.measure_space_def
+/-!
+# Tactics for measure theory
+
+Currently we have one domain-specific tactic for measure theory: `measurability`.
+
+This tactic is to a large extent a copy of the `continuity` tactic by Reid Barton.
+-/
+
+/-!
+### `measurability` tactic
+
+Automatically solve goals of the form `measurable f`, `ae_measurable f μ` and `measurable_set s`.
+
+Mark lemmas with `@[measurability]` to add them to the set of lemmas
+used by `measurability`. Note: `to_additive` doesn't know yet how to
+copy the attribute to the additive version.
```

Comment:
This seems like something that should be fixed before merging

Reply:
Do you know how I would do that? I would very much like to do it. And then I'll make another PR to do it for the `continuity` tactic as well.

Reply:
Do I just add `measurability` to the list of names line 312 of algebra/group/to_additive.lean? I'll experiment with that file tomorrow and see if I can make it work.

Reply:
`to_additive` doesn't do this for `continuity` either, fwiw

Reply:
This is easily fixed after #5487 and need not hold up this PR.